tables/table.py

```diff
@@ -1730,7 +1804,7 @@ def iterrows(self, start=None, stop=None, step=None):
         row = tableextension.Row(self)
         return row._iter(start, stop, step)
 
-    def __iter__(self):
+    def __iter__(self) -> Iterator:
```

Comment:
Here I guess it should be something like `Iterator[Row]` or something even more concrete.

Reply:
Ah, that is right. I made the annotation more explicit as `Iterator[tableextension.Row]`.
Thanks!